src/FactionMasterInvitationImprove.php

```diff
@@ -60,13 +60,15 @@ public function onLoad(): void {
 		$this->saveResource('en_EN.yml');
 		$this->saveResource('tr_TR.yml');
 		$this->saveResource('pt_BR.yml');
+                $this->saveResource('es_ES.yml');
 		$this->saveResource('ru_RU.yml');
 		$this->saveResource('config.yml');
 		ConfigUpdater::checkUpdate($this, $this->getConfig(), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "fr_FR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "en_EN.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "tr_TR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "pt_BR.yml", Config::YAML), "file-version", 1);
+                ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "es_ES.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "ru_RU.yml", Config::YAML), "file-version", 1);
 		$this->LangConfig = [
 			"FR" => new Config($this->getDataFolder() . "fr_FR.yml", Config::YAML),
```

Comment:
You need to specify in this array the Spanish language or it will not be taken into account when loading the extension

Reply:
Im blind sorry i will fix that